docs/docs/installation/configuring-superset.mdx

```diff
@@ -276,12 +276,22 @@ A current list of feature flags can be found in [RESOURCES/FEATURE_FLAGS.md](htt
 
 ### SECRET_KEY Rotation
 
-If you want to rotate the SECRET_KEY(change the existing secret key), follow the below steps.
-
-Add the new SECRET_KEY and PREVIOUS_SECRET_KEY to `superset_config.py`:
-
+To change the SECRET_KEY, first run the superset, because we need to access the database. After the superset running, create `superset_config.py` on temporary folder, for example `/tmp/superset_config.py` with following content
```

Comment:
any reason to use a temp superset_config.py instead of the actual superset_config.py?